src/www/services_rfc2136_edit.php

```diff
@@ -58,6 +58,9 @@
     $pconfig['usetcp'] = isset($a_rfc2136[$id]['usetcp']);
     $pconfig['usepublicip'] = isset($a_rfc2136[$id]['usepublicip']);
 
+    $pconfig['recordtype'] = $a_rfc2136[$id]['recordtype'];
+    if (!$pconfig['recordtype']) $pconfig['recordtype'] = "both";
```

Comment:
From a working OPNsense VM you can install the repository and upgrade from there:

    # opnsense-code -a wevcode opnsense-core
    # cd /usr/opnsense-core
    # git checkout rfc2136-recordtype-option
    # opnsense-update -t opnsense-devel
    # make upgrade